src/dx_azure_iot.c

```diff
@@ -23,6 +23,11 @@ static const char *_pnpModelId = NULL;
 static const char *_pnpModelIdJsonTemplate = "{\"modelId\":\"%s\"}";
 
 static IOTHUBMESSAGE_DISPOSITION_RESULT (*_messageReceivedCallback)(IOTHUB_MESSAGE_HANDLE message, void *context) = NULL;
+static void (*_deviceTwinCallbackHandler)(DEVICE_TWIN_UPDATE_STATE updateState, const unsigned char *payload, size_t payloadSize,
```

Comment:
Just a thought on these new methods to register callbacks.  I like the consistency with the recent C2D message callbacks. Did you consider allowing multiple callback registrations like we did with the C2D messages?  I'm not sure I can think of a good reason to register multiple twin or direct method callbacks, but it would add the flexibility if a user found a reason to use it.

Reply:
Likewise can't think of a good reason - but likewise agree consistency is good. I'll add.